web-registry/src/components/organisms/BasketOverview.tsx

```diff
@@ -0,0 +1,268 @@
+  content: {
+    display: 'flex',
+    width: '100%',
+    padding: theme.spacing(0, 5),
+    [theme.breakpoints.up('sm')]: {
+      padding: theme.spacing(30, 5),
+      maxWidth: theme.breakpoints.values.lg,
+    },
+    [theme.breakpoints.down('lg')]: {
+      flexDirection: 'column',
+    },
+  },
+  imageContainer: {
+    position: 'relative',
+    width: '100%',
+    [theme.breakpoints.down('sm')]: {
+      height: theme.spacing(57.5),
+      overflowY: 'clip',
+    },
+  },
+  image: {
+    width: '100%',
+    maxWidth: theme.spacing(103),
+    [theme.breakpoints.down('sm')]: {
+      height: theme.spacing(76.4),
+      minWidth: theme.spacing(103),
+      overflow: 'hidden',
+      position: 'absolute',
+      top: '50%',
+      left: '50%',
+      transform: 'translate(-50%, -50%)',
+    },
+  },
+  textContainer: {
+    display: 'flex',
+    flex: 1,
+    flexDirection: 'column',
+    [theme.breakpoints.down('lg')]: {
+      paddingLeft: theme.spacing(8),
+    },
+    [theme.breakpoints.down('sm')]: {
+      width: '100%',
+      padding: theme.spacing(8, 0),
+      '&:last-child': {
+        paddingBottom: theme.spacing(15),
+      },
+    },
+  },
+  title: {
+    lineHeight: theme.spacing(9),
+    marginBottom: theme.spacing(2),
+    [theme.breakpoints.down('sm')]: {
+      marginBottom: theme.spacing(0),
+    },
+  },
+  basketDenom: {
+    fontWeight: 'bold',
+    marginTop: theme.spacing(2),
+    color: theme.palette.info.main,
+  },
+  basketDescription: {
+    fontSize: theme.typography.pxToRem(22),
+    marginTop: theme.spacing(2),
+    [theme.breakpoints.down('md')]: {
+      fontSize: theme.typography.pxToRem(18),
+    },
+  },
+  card: {
+    margin: theme.spacing(8, 0, 0),
+    padding: theme.spacing(3, 5, 7),
+  },
+}));
+
+export interface BasketOverviewProps {
+  name: string;
+  basketDenom: string;
+  description: string;
+  totalAmount: number;
+  curator: string;
+  allowedCreditClasses: string[];
+  minStartDate?: string;
+  startDateWindow?: string;
+}
+
+const BasketOverview: React.FC<BasketOverviewProps> = ({
+  name,
+  basketDenom,
+  description,
+  totalAmount,
+  curator,
+  allowedCreditClasses,
+  minStartDate,
+  startDateWindow,
+}) => {
+  const styles = useStyles();
+
+  return (
+    <Box
+      sx={theme => ({
+        display: 'flex',
+        justifyContent: 'center',
+        borderBottom: `1px solid ${theme.palette.grey[100]}`,
+        backgroundColor: theme.palette.grey[50],
+        backgroundImage: `url(${topoImg})`,
+        backgroundSize: 'cover',
+        backgroundPosition: 'center',
+      })}
+    >
+      <Section className={styles.content}>
+        <Grid container>
+          <Grid item xs={12} sm={5} className={styles.imageContainer}>
+            <OptimizedImage
+              className={styles.image}
+              src={forestImg}
+              alt={name}
+            />
+          </Grid>
+          <Grid item xs={12} sm={7} className={styles.textContainer}>
```

Comment:
looks like many of these `makeStyles` classes aren't necessary. 

Could either handle with `sx` (which might take some reworking because of how `sx` handles breakpoints - they're all 'up' e.g. `<Grid sx={{ p: { xs: 10, sm: 12 } }}>` translates to `[theme.breakpoints.up('xs')]: padding: theme.spacing(10)` etc), or by making these with `styled`:
```tsx
const TextContainer = styled(Grid)(({ theme }) => ({
  display: 'flex',
  flex: 1,
  flexDirection: 'column',
  [theme.breakpoints.down('lg')]: {
    paddingLeft: theme.spacing(8),
  },
  [theme.breakpoints.down('sm')]: {
    width: '100%',
    padding: theme.spacing(8, 0),
    '&:last-child': {
      paddingBottom: theme.spacing(15),
    },
  },
}));
```

then you'd instantiate it as
```suggestion
          <TextContainer item xs={12} sm={7}>
```
etc